instrumentation/opentelemetry-instrumentation-tornado/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## Unreleased
+
+- Initial release. Supports Tornado 4.x and 5.x on Python 3.7 and newer.
```

Comment:
This says tornado 4/5 but the PR title calls for tornado 6+

Reply:
Yea, it's 6+. Need to fix that.